drivers/tty/serial/serial_core.c

```diff
@@ -3470,6 +3491,10 @@ void serial_core_unregister_port(struct uart_driver *drv, struct uart_port *port
 	if (!serial_core_ctrl_find(drv, phys_dev, ctrl_id))
 		serial_base_ctrl_device_remove(ctrl_dev);
 
+	/* Delete the anon platform_device */
```

Comment:
Same nitpick as above, the comment is not very useful, remove.

Reply:
Comment removed